lib/query/querybuilder.js

```diff
@@ -148,6 +148,25 @@ class Builder extends EventEmitter {
     return this;
   }
 
+  // Adds a one or more optimizer hits to the list of "optimizerHints" on the query.
+  optimizerHint() {
+    const hints = helpers.normalizeArr.apply(null, arguments);
+    if (hints.some((hint) => !isString(hint))) {
+      throw new Error('Optimizer hint must be a string');
+    }
+    if (hints.some((hint) => hint.includes('/*') || hint.includes('*/'))) {
+      throw new Error('Optimizer hint cannot include "/*" or "*/"');
+    }
+    if (hints.some((hint) => hint.includes('?'))) {
+      throw new Error('Optimizer hint cannot include "?"');
```

Comment:
If there is `?` in the hint it messes up parameter bindings. So I'm forbidding `?` symbols in hints. Later we can somehow  escape `?` in hints or actually allow parameter bindings in hints if they are supported by any dialect.